features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Base/RouteMenuMapping.jsx

```diff
@@ -106,6 +107,15 @@ const RouteMenuMapping = (intl) => [
         component: () => <MicrogatewayLabels />,
         icon: <PhonelinkSetupIcon />,
     },
+    {
+        id: intl.formatMessage({
+            id: 'Base.RouteMenuMapping.keymanagers',
+            defaultMessage: 'KeyManagers',
+        }),
+        path: '/settings/key-managers',
+        component: () => <KeyManagers />,
```

Comment:
In the component prop , we can just pass the component variable, no need to wrap it with a function(lambda)